src/ray/raylet/node_manager.h

```diff
@@ -540,6 +542,12 @@ class NodeManager : public rpc::NodeManagerServiceHandler {
   /// unable to schedule new tasks or actors at all.
   void WarnResourceDeadlock();
 
+  /// Move tasks from runnable -> running as workers become available.
+  void DispatchDirectCallTasks();
+
+  /// Move tasks from pending -> runnable queue as resources become available.
+  void NewSchedulerScheduleMoreTasks();
```

Comment:
Consider `NewSchedulerSchedulePendingTasks()`, and updating the comment.

Reply:
done